server/src/main/java/com/workshop/domain/Status.java

```diff
@@ -0,0 +1,8 @@
+package com.workshop.domain;
+
+public enum Status {
+    OPEN,
+    PROGRESS,
```

Comment:
Maybe IN_PROGRESS would be more adequate, but I'm just picky here :D

Reply:
I've also considered this but decided to follow notes from the call. So changing it to IN_PROGRESS